external/cmake-files/README_SuiteSparse.md

```diff
@@ -0,0 +1,65 @@
+# SuiteSparse Integration
+
+This directory contains CMake files for downloading and building external dependencies, including SuiteSparse.
+
+## SuiteSparse
+
+The `SuiteSparse.cmake` file handles the automatic download, build, and configuration of SuiteSparse from the official GitHub repository.
+
+### Components Built
+
+The following SuiteSparse components are built:
+- **SuiteSparse_config**: Core configuration
+- **AMD**: Approximate Minimum Degree ordering
+- **COLAMD**: Column Approximate Minimum Degree ordering  
+- **CAMD**: Constrained Approximate Minimum Degree ordering
+- **CCOLAMD**: Constrained Column Approximate Minimum Degree ordering
+- **CHOLMOD**: Sparse Cholesky factorization
+- **SPQR**: Sparse QR factorization
+
+### CMake Integration
+
+The SuiteSparse library is fetched and built automatically when you configure the project:
+
+```bash
+cd test
+mkdir build
+cd build
+cmake ..
+make
+```
+
+### Dependencies
+
+SuiteSparse requires:
+- BLAS (Basic Linear Algebra Subprograms)
+- LAPACK (Linear Algebra Package)
+
+These are typically available on most systems:
+- **macOS**: Pre-installed as part of Accelerate framework
+- **Linux**: Install via `libblas-dev` and `liblapack-dev` packages
+
+The CMake build system will automatically find these dependencies.
+
+### Commit Version
+
+The integration uses SuiteSparse commit: `d558c83006d63d1dc62004f30042b3ca484f3f94`
```

Comment:
Why not a stable release? eg. https://github.com/DrTimothyAldenDavis/SuiteSparse/tree/v7.12.1